user/views.py

```diff
@@ -1,18 +1,17 @@
-from datetime import datetime, timedelta
-from json     import JSONDecodeError
-from jwt      import DecodeError
+from datetime           import datetime, timedelta
+from json               import JSONDecodeError
+from jwt                import DecodeError
 
-import bcrypt
-import json
-import jwt
-import re
+import bcrypt, json, jwt, re
```

Comment:
```suggestion
import (
    jwt,
    bcrypt,
    joson,
    re
)
```
위와 같은 형식이 컨벤션 입니다. 
수정해 주세요.